.github/workflows/prof_correctness.yml

```diff
@@ -0,0 +1,70 @@
+name: Profiling correctness
+
+on:
+  - pull_request
+
+jobs:
+  prof-correctness:
+    runs-on: ubuntu-latest
+
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v2
+        with:
+          fetch-depth: 0
+          submodules: true
+
+      - name: Setup PHP
+        uses: shivammathur/setup-php@v2
+        with:
+          php-version: '8.2'
+
+      - name: Restore cached Primes
```

Comment:
Cached primes? Is this a term I don't know or is this a leftover from copy/paste?